src/main/java/it/pagopa/transactions/utils/JwtTokenUtils.java

```diff
@@ -23,26 +24,26 @@ public class JwtTokenUtils {
 
     public static final String TRANSACTION_ID_CLAIM = "transactionId";
 
-    private final int tokenValidityTimeMillis;
+    private final int tokenValidityTimeSecond;
 
     public JwtTokenUtils(
             @Autowired SecretKey jwtSecretKey,
-            @Value("${jwt.validityMillis}") int tokenValidityMillis
+            @Value("${payment.token.validity}") int tokenValiditySecond
     ) {
         this.jwtSecretKey = jwtSecretKey;
-        this.tokenValidityTimeMillis = tokenValidityMillis;
+        this.tokenValidityTimeSecond = tokenValiditySecond;
     }
 
     public Mono<String> generateToken(TransactionId transactionId) {
         try {
             Calendar calendar = Calendar.getInstance();
             Date issuedAtDate = calendar.getTime();
-            calendar.add(Calendar.MILLISECOND, tokenValidityTimeMillis);
+            calendar.add(Calendar.MILLISECOND, (int) Duration.ofSeconds(tokenValidityTimeSecond).toMillis());
```

Comment:
Is it different from
```suggestion
            calendar.add(Calendar.SECOND, tokenValidityTimeSecond);
```

Reply:
OK you’re right

Reply:
Fixed with 23d0ce22fb07b52d820ce166e176eb199f18a92c